cmd/maya-apiserver/app/server/http.go

```diff
@@ -265,8 +265,8 @@ func (s *HTTPServer) registerHandlers(serviceProvider string, enableDebug bool)
 		latestOpenEBSVolumeRequestDuration, s.volumeSpecificRequest))
 
 	// Request w.r.t to a single VSM entity is handled here
-	s.mux.HandleFunc("/v1alpha1/volumes/", s.wrap(openEBSVolumeRequestCounterV1alpha1,
-		openEBSVolumeRequestDurationV1alpha1, s.volumeV1alpha1SpecificRequest))
+	//s.mux.HandleFunc("/v1alpha1/volumes/", s.wrap(openEBSVolumeRequestCounterV1alpha1,
+	//	openEBSVolumeRequestDurationV1alpha1, s.volumeV1alpha1SpecificRequest))
```

Comment:
Should we just complete remove these lines? Or is the idea that idea that versioning will be re-introduced later.

Reply:
I have kept this and other commented lines.
Some are kept to refactor while some needs a good thought before removing completely.
I was not able to put TODO at these places.

I will re-read the PR and remove some of them or add TODO if required.